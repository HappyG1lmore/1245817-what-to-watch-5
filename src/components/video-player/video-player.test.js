import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import VideoPlayer from './video-player';

const renderer = new ShallowRenderer();

test(`VideoPlayer renders correctly`, () => {
  renderer.render(
      <VideoPlayer
        playbackActive={false}
        poster="/poster.jpeg"
        src="/video.mp4"
      />
  );
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
