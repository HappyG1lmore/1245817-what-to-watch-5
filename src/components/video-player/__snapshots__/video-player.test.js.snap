// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer renders correctly 1`] = `
<video
  poster="/poster.jpeg"
  src="/video.mp4"
/>
`;
